content/installation/java/installation/MavenCargoPlugin.md

```diff
@@ -5,27 +5,25 @@ tags: "java agent installation maven codehaus"
 -->
 
 
-## Prerequisites
```

Comment:
This is a prerequisite for all of these examples, but not all of them specify this. I thought it was better to leave it out: I don't want to encourage users to "Download the Contrast plugin (***contrast.jar***) from the Contrast site"